### DISCUSSION TITLE
Skip chrome history

### DISCUSSION THREAD

**Discussion by csm10495** [aeharding/voyager#141] [Q&A]
On Android, is there a way to make it so everything browsed in wefwef doesn't show up in Chrome history? 

I like the idea of it being it's own isolated app as opposed to polluting my chrome history.

**Comment by aeharding** [comment]
If you add to homescreen, does it show up in history?

**Comment by csm10495** [comment]
Yep, still does (which surprised me)

![Screenshot_20230701-115610.png](https://github.com/aeharding/wefwef/assets/5749838/1d924c21-a990-47dc-b60f-0fe9dab8189f)

After tapping around a bit, chrome history looks like ^

**Reply by aeharding** [reply]
Hmmm. I'm not sure why that's happening, or if there's a good way to prevent it. I don't think it's a wefwef bug, however.

I'd love to hear what anyone more familiar with android PWAs has to say on this.

**Comment by spacemelon1** [comment]
Could it be due to web scraping it shows up on your browser history?

**Reply by aeharding** [reply]
This is an Android quirk. Worth noting that this does not happen on iOS.

https://bugs.chromium.org/p/chromium/issues/detail?id=1327165

**Comment by Charlotte-br560** [comment]
Currently, there isn't a direct way to prevent wefwef browsing from appearing in Chrome history on Android. However, one workaround could be to use a separate browser solely for wefwef, keeping your Chrome history clean.
